grails-app/domain/org/pih/warehouse/product/ProductSupplier.groovy

```diff
@@ -107,7 +110,7 @@ class ProductSupplier implements Serializable, Comparable<ProductSupplier> {
         active(nullable: true)
     }
 
-    ProductPackage getDefaultProductPackage() {
+    ProductPackage getDefaultProductPackageDerived() {
         return productPackages ? productPackages.sort { it.lastUpdated }.last() : null
```

Comment:
maybe it is worth to create `compareTo` in the `ProductPackage` domain

Reply:
I just renamed the method, didn’t care about the content, because it was not in the scope of my work

Reply:
Oh, that's true, didn't notice that, sry